### PR TITLE
Create ProyectoIntegrAccion.json

### PR DESCRIPTION
**Pull request by Proyecto-IntegrAccion**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6739"></a>

### CODE REVIEW

File: ProyectoIntegrAccion.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Proyecto IntegrAcción",
+  "type": "comunidad",
+  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR"
+{
```
Comment (kumarsonsoff3) [inline ProyectoIntegrAccion.json:5]: There us a missing `links` key. Please make sure you follow the [quickstart guide](https://linkfree.io/docs/quickstart). 

```suggestion
"links": [
    {
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your file should be in data directory. Please rename your [file here](https://github.com/Proyecto-IntegrAccion/LinkFree/edit/main/ProyectoIntegrAccion.json) to `data/Proyecto-IntegrAccion.json`. 

Change `ProyectoIntegrAccion.json`  ->  `data/Proyecto-IntegrAccion.json`. Let us know if you need any help with this.

File: ProyectoIntegrAccion.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "name": "Proyecto IntegrAcción",
+  "type": "comunidad",
```
Comment (SaraJaoude) [inline ProyectoIntegrAccion.json:3]: currently we only support `personal`/actualmente solo apoyamos `personal`:
```suggestion
  "type": "personal",
```

File: ProyectoIntegrAccion.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Proyecto IntegrAcción",
+  "type": "personal",
+  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR"
```
Comment (SaraJaoude) [inline ProyectoIntegrAccion.json:4]: a comma is necessary / una coma es necesaria
```suggestion
  "bio": "un proyecto de cooperación entre la Universidad Católica San Pablo y ACNUR",
```

**Review by Pradumnasaraf — APPROVED** [review]
Hey everyone,

Thank you all for the review. Since the pull request has been open for a while, I will be merging it into a temporary branch to address any existing errors before merging it into the `main` branch.

**Pull request merged**